### ISSUE TITLE
Date picker: Display the word "Today" on today's day

### ISSUE THREAD

**Issue by Kiarokh** [Lundalogik/lime-elements#1896]
## New feature motivation
<img width="666" alt="image" src="https://user-images.githubusercontent.com/35954987/196684615-80442658-442f-498a-9209-586578a278a9.png">

When a date picker is opened, it's hard to understand what the round round around one of the numbers mean.
That means "Today", but that's not enough, and not accessible.

## New feature description

https://user-images.githubusercontent.com/35954987/196684496-6cfab221-c5ac-4ae2-8859-bf581b3d14f6.mov

## New feature implementation
Using the [`attr` CSS function](https://developer.mozilla.org/en-US/docs/Web/CSS/attr), we could display the Today's title.
